### PR TITLE
additional labels for eventing-contrib

### PR DESCRIPTION
**Pull request by lionelvillard**
**What this PR does, why we need it**:
it has additional labels for categorizing eventing-contrib issues by channel and source type.

**Which issue(s) this PR fixes**:

**Special notes to reviewers**:

**User-visible changes in this PR**:

### CODE REVIEW

**Comment by chizhg** [comment]
Hi @lionelvillard , this change needs to be made in `config/prow/templates/prow_plugins.yaml`, and then you'll need to run `make -C config/prow config` to generate the config file.

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -74,6 +74,15 @@ label:
   - proposal/0.17
   - proposal/0.18
   - proposal/0.19
+  - channel/kafka
```
Comment (chizhg) [inline config/prow/plugins.yaml]: It looks Prow does not support setting additional_labels for a repo... Can you add a comment here to say these labels are specific to eventing?

Comment (lionelvillard) [reply config/prow/plugins.yaml]: done

**Pull request merged**